scripts/setup-dev.js

```diff
@@ -0,0 +1,24 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+const fs = require('fs-extra');
+const path = require('path');
+
+const target = path.resolve('../vets-website/build/localhost/generated');
```

Comment:
It does not, path.resolve in this case starts in the working directory where the script is run, not where the script is located.  The symLinkPath only needs it because the reference to __dirname